packages/interchain-token-service-api/src/client.rs

```diff
@@ -0,0 +1,31 @@
+use cosmwasm_std::{to_json_binary, Addr, WasmMsg};
+
+use crate::msg::ExecuteMsg;
+use crate::Message;
+
+pub struct Router {
+    pub address: Addr,
+}
+
+impl Router {
+    fn execute(&self, msg: &ExecuteMsg) -> WasmMsg {
+        WasmMsg::Execute {
+            contract_addr: self.address.to_string(),
+            msg: to_json_binary(&msg).expect("msg should always be serializable"),
+            funds: vec![],
+        }
+    }
+
+    pub fn route(&self, msgs: Vec<Message>) -> Option<WasmMsg> {
+        ignore_empty(msgs).map(|msgs| self.execute(&ExecuteMsg::RouteMessages(msgs)))
+    }
+}
```

Comment:
This looks like a client for the router. Why is this here?

Reply:
same, got checked in by mistake